src/log4net/Core/LoggingEvent.cs

```diff
@@ -742,15 +742,27 @@ private static string ReviseThreadName(string? threadName)
   {
     if (_platformDoesNotSupportWindowsIdentity)
     {
-      // we've already received one PlatformNotSupportedException
+      // we've already received one PlatformNotSupportedException or null from TryReadWindowsIdentityUserName
```

Comment:
Can you fix the comment on UserName (especially clarify the "NOT AVAILABLE" part)?

Reply:
I will fix it.